### PR TITLE
Use svobs mode in all website deployment config

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/configmodule.py [hunk 1]
```diff
@@ -11,10 +11,6 @@ class Config:
     WEBDRIVER = False
     LOCAL = False
     LITE = False
-    # Use StatVarObs instead of PopObs data model.
-    # TODO(shifucun): change this to True after setting svobs_mode=true in mixer
-    # by default.
-    SVOBS = False
```
Comment (beets) [inline server/configmodule.py:17]: Should it be set to True in the default config? There are still a few more references to this var around (e.g. kg_main.html). 

Let's flip it to True by default first, and clean up by deleting it in a follow up (similar to the protocol for an experiment launch).

File: server/configmodule.py [hunk 2]
```diff
@@ -40,11 +36,11 @@ class AutopushConfig(Config):
 
 
 class SvObsConfig(Config):
```
Comment (beets) [inline server/configmodule.py:38]: Is this config class still required?

Comment (shifucun) [reply server/configmodule.py:39]: This is for svobs.datacommons.org. Can remove this with the flag after migration?

File: server/configmodule.py (hunk 1 above)
Comment (shifucun) [reply server/configmodule.py:17]: Good catch. Set it to True now

File: server/configmodule.py (hunk 2 above)
Comment (beets) [reply server/configmodule.py:39]: yup, sg

**Review by beets — APPROVED** [review]

**Pull request merged**